csaf-validation/src/main/kotlin/io/github/csaf/sbom/validation/Requirement.kt

```diff
@@ -122,11 +136,11 @@ operator fun Requirement.plus(other: Requirement): Requirement {
     return And(this, other)
 }
 
-internal class And(var lhs: Requirement, var rhs: Requirement) : Requirement {
+internal class And(private val lhs: Requirement, private val rhs: Requirement) : Requirement {
```

Comment:
Documentation. Somehow redundant to `AllOf` but that's closer to the standard I guess. Maybe share the implementation?

Reply:
similar to above